promtail/imageroot/promtail.service

```
@@ -0,0 +1,31 @@
+[Unit]
+Description=Promtail logs collector for Loki
+
+[Service]
+Type=forking
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/var/lib/nethserver/%N/state/environment
+Restart=on-failure
+ExecStartPre=/bin/rm -f %t/%N.pid %t/%N.cid
+ExecStart=/usr/bin/podman run \
+    --conmon-pidfile %t/%N.pid \
+    --cidfile %t/%N.cid \
+    --cgroups=no-conmon \
+    --replace \
+    --log-opt=tag=promtail \
+    --name %N \
+    -d \
+    -e LOKI_URL=${LOKI_URL} \
+    -v /var/log/journal/:/var/log/journal/ \
```

This might alter the SELinux labels: did you check if the default label is retained?

```suggestion
    -v /var/log/journal/:/var/log/journal/ \
```

---

I don't know what you mean, I have only followed the documentation: https://grafana.com/docs/loki/latest/clients/promtail/scraping/#journal-scraping-linux-only

How I can check it?

---

restorecon -v -n /var/log/journal

---

I think is ok.
```
root@leader:/home/vagrant# restorecon -R -v -n /var/log/journal
root@leader:/home/vagrant# echo $?
0
```